README.md

```diff
@@ -6399,6 +6408,18 @@ class Bitcoin(Account):
         self.balance = balance
 
 
+# ----------------------------
```

Comment:
Hey there, it looks like there’s a duplication in this section. Could you take a look?